docs/patterns/components/Alert/index.js

```diff
@@ -26,32 +26,46 @@ export default class Demo extends Component {
         <Button onClick={this.toggleDefaultAlert}>Default Alert</Button>
         <Button onClick={this.toggleWarningAlert}>Warning Alert</Button>
         <Alert heading="Default Alert" show={showDefaultAlert}>
-          <AlertContent>Dismissable alert</AlertContent>
+          <AlertContent>
```

Comment:
> Alerts never have inputs

I have to disagree with that statement. Isn't our "Save you test" dialog an alert? This seems like a weirdly strict limitation to put on a component in a pattern library.


<img width="470" alt="Screen Shot 2021-12-20 at 9 18 21 PM" src="https://user-images.githubusercontent.com/3180185/146875469-58ad75bc-6d9a-4645-b5b3-ab1eaae9950e.png">